ENGAGEHF/Dashboard/Vitals/Views/VitalsCard.swift

```diff
@@ -0,0 +1,109 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import HealthKit
+import SwiftUI
+
+
+struct VitalsCard: View {
+    @ScaledMetric private var quantityTextSize: CGFloat = 20
+    
+    let quantity: String?
+    let units: String?
+    let type: String?
+    let date: Date?
+    
+    private var displayDate: String? {
+        date?.formatted(date: .numeric, time: .omitted)
+    }
+    
+    
+    var body: some View {
+        StudyApplicationListCard {
+            VStack(alignment: .center, spacing: 10) {
+                Text(type ?? "--")
+                    .font(.headline)
+                    .foregroundColor(.primary)
+                    .multilineTextAlignment(.center)
+                HStack {
+                    Text(quantity ?? "--")
+                        .font(.system(size: quantityTextSize, weight: .semibold, design: .rounded))
+                        .multilineTextAlignment(.center)
+                        .accessibilityLabel("\(type ?? "--") Quantity: \(quantity ?? "--")")
+                    Text(units ?? "--")
+                        .font(.subheadline)
+                        .foregroundStyle(Color.secondary)
+                        .multilineTextAlignment(.center)
+                        .accessibilityLabel("\(type ?? "--") Units: \(units ?? "--")")
+                }
+                Text(displayDate ?? "--")
```

Comment:
I would suggest a smaller font here that could allow us to display the date and time:
```suggestion
                Text(displayDate ?? "--")
                    .font(.caption)
                    .foregroundColor(.secondary)
```